gir/pkgconfig/pkgconfig.go

```diff
@@ -1,105 +1,152 @@
+// Copyright (C) 2021, 2023  diamondburned
+// Copyright (C) 2023  Luke Shumaker
+
 // Package pkgconfig provides a wrapper around the pkg-config binary.
 package pkgconfig
 
 import (
 	"errors"
 	"fmt"
-	"io/fs"
-	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
-	"sync"
-)
-
-var (
-	path     string
-	pathErr  error
-	pathOnce sync.Once
 )
 
-func ensure() error {
-	pathOnce.Do(func() {
-		path, pathErr = exec.LookPath("pkg-config")
-		if pathErr != nil {
-			pathErr = fmt.Errorf("failed to find pkg-config: %w", pathErr)
-		}
-	})
-	return pathErr
-}
-
-// IncludeDir finds the include directories for the given packages.
-func IncludeDirs(pkgs ...string) ([]string, error) {
-	if err := ensure(); err != nil {
-		return nil, err
+// Variable returns a map of map[pkg]value containing the given
```

Comment:
It should be "returns a map of pkg to value containing..." or "returns a map[pkg]value containing..."

Reply:
I've reworded all of the doc comments to have more consistent wording.